src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.tsx

```diff
@@ -96,6 +107,7 @@ export const ScoreContent = ({
     name: scoreConditionalsName,
     keyName: REACT_HOOK_FORM_KEY_NAME,
   });
+  const conditionalsSize = scoreConditionals?.length ?? 0;
```

Comment:
```suggestion
  const conditionalsSize = scoreConditionals?.length ?? null;
```

Reply:
No need, since we use this value later in comparison